pimd/pim_nb_config.c

```diff
@@ -2229,6 +2229,11 @@ int lib_interface_pim_address_family_mroute_destroy(
 		yang_dnode_get_ip(&source_addr, args->dnode, "./source-addr");
 		yang_dnode_get_ip(&group_addr, args->dnode, "./group-addr");
 
+#if PIM_IPV == 6
+		/*TBD
+		  pimv6_static_del();*/
+		break;
+#endif
```

Comment:
The remaining code will get executed for both ipv4 and ipv6. Check this for all the commits.

Reply:
I have break the condition, so it should come out of switch